src/test/scala/com/cognite/sdk/scala/common/WritableBehaviors.scala

```diff
@@ -90,65 +106,95 @@ trait WritableBehaviors extends Matchers { this: FlatSpec =>
+          (sameMissingIds should contain).theSameElementsAs(sameIdsThatDoNotExist.toSet)
         }
-        sameMissingIds should have size sameIdsThatDoNotExist.toSet.size.toLong
-        sameMissingIds should contain theSameElementsAs sameIdsThatDoNotExist.toSet
       }
     }
 
     it should "create and delete items using the read class and external ids" in {
       // create a single item
       val createdItem = writable.createFromRead(readExamples.take(1))
-      createdItem should have size 1
+      (createdItem should have).size(1)
       createdItem.head.externalId should not be empty
-      writable.deleteByExternalIds(createdItem.map(_.externalId.get))
+
+      if (deleteFlag)
+        writable.deleteByExternalIds(createdItem.map(_.externalId.get))
 
       // create multiple items
-      val createdItems = writable.createFromRead(readExamples)
-      createdItems should have size readExamples.size.toLong
-      val createdExternalIds = createdItems.map(_.externalId.get)
-      createdExternalIds should have size readExamples.size.toLong
-      writable.deleteByExternalIds(createdExternalIds)
+      if (deleteFlag) {
+        val createdItems = writable.createFromRead(readExamples)
+        (createdItems should have).size(readExamples.size.toLong)
+        val createdExternalIds = createdItems.map(_.externalId.get)
+        (createdExternalIds should have).size(readExamples.size.toLong)
+        writable.deleteByExternalIds(createdExternalIds)
+      }
+      else {
+        val createdItems = writable.createFromRead(Seq(readExamples(1),readExamples(2)))
+        (createdItems should have).size(2)
+        val createdExternalIds = createdItems.map(_.externalId.get)
+        (createdExternalIds should have).size(2)
+      }
     }
 
+
     it should "create and delete items using the create class and external ids" in {
       // create a single item
       val createdItem = writable.create(createExamples.take(1))
-      createdItem should have size 1
+      (createdItem should have).size(1)
       createdItem.head.externalId should not be empty
-      writable.deleteByExternalIds(createdItem.map(_.externalId.get))
+
+      if (deleteFlag)
+        writable.deleteByExternalIds(createdItem.map(_.externalId.get))
 
       // create multiple items
-      val createdItems = writable.create(createExamples)
-      createdItems should have size createExamples.size.toLong
-      val createdIds = createdItems.map(_.externalId.get)
-      createdIds should have size createExamples.size.toLong
-      writable.deleteByExternalIds(createdIds)
+      if (deleteFlag) {
+        val createdItems = writable.create(createExamples)
+        (createdItems should have).size(createExamples.size.toLong)
+        val createdIds = createdItems.map(_.externalId.get)
+        (createdIds should have).size(createExamples.size.toLong)
+        writable.deleteByExternalIds(createdIds)
+      }
+      else {
```

Comment:
This will go away, but we always do `} else {`. We also always use curly braces for `if`,
```scala
if (deleteFlag) {
  ...
}
```
never this:
```scala
if (deleteFlag)
  ...
```